includes/admin/views/html-admin-page-status-tools.php

```diff
@@ -11,22 +11,46 @@
 	exit;
 }
 
+foreach ( $tools as $action_name => $tool ) {
+	// phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
+	?>
+	<form id="<?php echo 'form_' . $action_name; ?>" method="GET" action="<?php echo esc_url( admin_url( 'admin.php?foo=bar' ) ); ?>">
```

Comment:
Instead of ignoring the escape perhaps we should use `esc_attr()` here?

Reply:
I thought that was not necessary since all the attribute values are fixed and we know beforehand it's going to work and be secure. But sure, adding `esc_attr`s won't hurt.